macosx/CreatorWindowController.mm

```diff
@@ -360,7 +360,20 @@ - (IBAction)cancelCreateProgress:(id)sender
 
 - (IBAction)incrementOrDecrementPieceSize:(id)sender
 {
-    auto const piece_size = static_cast<uint32_t>(pow(2.0, [sender intValue]));
+    auto power = [(NSStepper*)sender intValue];
+    if (power >= 32)
+    {
+        // over max piece size
+        [(NSStepper*)sender setIntValue:31];
+        return;
+    }
+    if (power <= 13)
+    {
+        // below min piece size
+        [(NSStepper*)sender setIntValue:14];
+        return;
+    }
+    auto const piece_size = uint32_t(1) << power;
```

Comment:
nit: `uint32_t(1)` -> `1U`?

Reply:
OK. Can you give a second look now, @nevack ? :)